### PR TITLE
Fix local env issues

### PR DESCRIPTION
**Pull request by MrLotU**

### CODE REVIEW

File: web.Dockerfile [hunk 1]
```diff
@@ -15,14 +15,18 @@ RUN swift build -c release && mv `swift build -c release --show-bin-path` /build
...
   libicu55 libxml2 libbsd0 libcurl3 libatomic1 \
   tzdata \
   && rm -r /var/lib/apt/lists/*
 WORKDIR /app
 COPY --from=builder /build/bin/Run .
 COPY --from=builder /build/lib/* /usr/lib/
-COPY --from=builder /app/Public ./Public
-COPY --from=builder /app/Resources ./Resources
+# Uncomment the next line if you need to load resources from the `Public` directory
+#COPY --from=builder /app/Public ./Public
```
Comment (joscdk) [inline web.Dockerfile]: This shouldn't be commented in this template

File: web.Dockerfile [hunk 2]
```diff
@@ -15,14 +15,18 @@ RUN swift build -c release && mv `swift build -c release --show-bin-path` /build
...
+# Uncommand the next line if you are using Leaf
+#COPY --from=builder /app/Resources ./Resources
```
Comment (joscdk) [inline web.Dockerfile]: This shouldn't be commented in this template

**Review by joscdk — CHANGES_REQUESTED** [review]

**Review by joscdk — APPROVED** [review]
LGTM 👍

**Comment by penny-coin** [comment]
Hey @MrLotU, you just merged a pull request, have a coin! 

You now have 55 coins.

**Pull request merged**